packages/color-scale-block/src/settings.ts

```diff
@@ -39,6 +39,13 @@ export const settings: BlockSettings = {
                     defaultValue: '100px',
                     clearable: false,
                     rules: [numericalOrPixelRule],
+                    onChange: (bundle: Bundle): void => {
```

Comment:
```suggestion
                    onChange: (bundle) => {
```
Type are defined by `BlockSettings`